src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -575,15 +584,22 @@ def fuzz(self,
            additional_args=None,
            extra_env=None):
     """LibFuzzerCommon.fuzz override."""
-    self._bind_corpus_dirs(corpus_directories)
+    bind_directories = copy.copy(corpus_directories)
+    if artifact_prefix:
+      bind_directories.append(artifact_prefix)
+
+    self._bind_corpus_dirs(bind_directories)
     corpus_directories = self._get_chroot_corpus_paths(corpus_directories)
 
+    if artifact_prefix:
+      artifact_prefix = self._get_chroot_directory(artifact_prefix)
+
     # Set artifact prefix to '/' in minijail.
```

Comment:
Probably this comment should be removed.

Reply:
Ignore this is fixed in https://github.com/google/clusterfuzz/pull/963#pullrequestreview-286285871

Reply:
Ignore; this is fixed in 
https://github.com/google/clusterfuzz/pull/963#pullrequestreview-286285871